src/psfmachine/machine.py

```diff
@@ -849,6 +851,14 @@ def build_shape_model(self, plot=False, flux_cut_off=1):
 
         # We then build the same design matrix for all pixels with flux
         self._get_mean_model()
+        # remove background pixels and recreate mean model
+        self._remove_background_pixels(flux_cut_off=flux_cut_off)
+
+        if plot:
+            return self.plot_shape_model()
+        return
+
+    def _remove_background_pixels(self, flux_cut_off=1):
```

Comment:
Need docstring for what '1' is here. 1 count, 1%...? (I know that I wrote this 🤦‍♀️ )

Reply:
JMP: This function updates the source mask, not the `psf_w`

CH: In that case, it needs a new name

Reply:
Whole method needs a docstring